frontend/components/puzzle5.jsx

```diff
@@ -1,100 +1,16 @@
 import React from 'react';
-import styled from 'styled-components';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { smSpacing, mdSpacing, lgSpacing } from '../constants';
-import { MdSpacing } from '../wrappers';
 
-import PuzzleHeader from './puzzleHeader';
-
-const Container = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-`;
-
-const PanelContainer = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-  width: 100%
-`;
-
-const PanelItem = styled.div`
-  border: 1px solid black;
-  border-radius: ${lgSpacing}px;
-  padding: ${smSpacing}px;
-`;
-
-const YourSwitchContainer = styled.button`
-  padding: ${mdSpacing};
-`;
+import Submit from './submit';
```

Comment:
Puzzle 5 is the switches puzzle right?

Not sure how this puzzle will need AnswerAwareInput and a submit button

Reply:
Puzzle 5 is the 10x10 grid with the X's (see the images in this PR)

Reply:
ahh never the submit component makes sense then